src/components/Shared/FeedbackDialog.vue

```diff
@@ -26,36 +29,41 @@ import { Notify } from 'quasar'
 export default {
   mixins: [mixinGeneral],
   data(){
-      return{
-          feedback:'',
-
-      }
+    return{
+      feedback:'',
+      platformdata:{}
+    }
   },
-
-    methods:{
-
-   
+  methods:{
     sendFeedbackMethod(){
-            this.setItemAction({
-                data:{
-                    issue:this.feedback,
-                    date:this.timeStamp,
-                    user:this.myUserId
-                },
-                path:'Feedback/'+this.timeStamp
-            }) 
-            Notify.create({
-                type: 'positive',
-                color: 'positive',
-                timeout: 3000,
-                message: 'Thanks for your feedback! We will have a look at it!'
-            })
-            this.feedback=''
-            this.$emit('close')
+      this.setItemAction({
+        data:{
+            issue:this.feedback,
+            date:this.timeStamp,
+            user:this.myUserId,
+            platformdata:this.platformdata
         },
-    
+        path:'Feedback/'+this.timeStamp
+      }) 
+      Notify.create({
+          type: 'positive',
+          color: 'positive',
+          timeout: 3000,
+          message: 'Thanks for your feedback! We will have a look at it!'
+      })
+      this.feedback=''
+      this.$emit('close')
+    },
+  },
+  mounted(){
+    this.platformdata={
```

Comment:
Still needed or just debug code?

Reply:
This is new extra information that will show more information regarding the device data of the user who send feedback.